creativecloud/features/firefly/firefly-interactive.css

```diff
@@ -24,6 +24,10 @@
   display: flex;
 }
 
+.genfill-promptbar > button:hover {
+  cursor: pointer;
+}
+
 .interactive-marquee .media {
```

Comment:
Please use firefly specific classes here. I see that these styles are also getting applied to genfill variant.